README.md

```diff
@@ -1,2 +1,113 @@
-# Proof-of-Attendance
-Community Event Management Tool: Build a library to provide infrastructure that allows event organizers to mint and distribute Attendance NFTs on the XRP Ledger.
+# Attendify XRPL API library
+
+## Project information
+
+Project was firstly developed during [XRPL NFT hackathon.](https://devpost.com/software/xrp-nft-attendence)
```

Comment:
Could you include a 1-2 high level description of it at the top? (You can maybe borrow the preview one of "Attendify is an API to allow event organizers to mint and distribute NFTs on the XRP Ledger to track and reward attendance at their events."

That way when they get to the features, they have a big picture to attach the individual use cases to :)

Reply:
Added description, also the docs are now [hosted online just like you requested](https://justanotherdevv.github.io/AttendifyXRPL/Attendify.html).
I have requested review of changes.